js/main.js

```diff
@@ -219,10 +418,17 @@ function clearGallery() {
 function toggleDB(currentDB) {
   if (currentDB === 'Drinks') {
     $body.classList.replace('meal-background', 'drink-background');
-    for (var i = 0; i < $uniformColor.length; i++) {
+    for (var i = 0; i < $navSwappers.length; i++) {
+      $navSwappers[i].setAttribute('data-view', $navSwappers[i].getAttribute('data-view').replace('Meals', 'Drinks'));
+      if ($navSwappers[i].nodeName === 'OPTION') {
+        $navSwappers[i].setAttribute('value', $navSwappers[i].getAttribute('value').replace('Meals', 'Drinks'));
+      }
+    }
```

Comment:
You've repeated this set of actions a couple times in your code, this would be a good opportunity for a function that takes the two data values to swap as arguments